frontend/src/components/survey-item/survey-item.tsx

```diff
@@ -50,6 +53,19 @@ export const SurveyItem = (props: SurveyItemProps) => {
     return images[Math.floor(Math.random() * images.length)];
   };
 
+  const getStatusColor = () => {
+    switch (status) {
+      case AnswerSurveyStatus.Completed:
+        return 'green';
+      case AnswerSurveyStatus.Started:
+        return 'yellow';
+      case AnswerSurveyStatus.NotStarted:
+        return 'red';
+      default:
+        return 'gray';
+    }
```

Comment:
This can be improved with object literal.

Reply:
Good idea!